wasm/src/project_system.rs

```diff
@@ -187,14 +188,41 @@ pub(crate) fn get_manifest_transformer(js_val: JsValue, _: String) -> Option<Man
 
     };
 
+    let lints: Vec<LintConfig> = match js_sys::Reflect::get(&js_val, &JsValue::from_str("lints")) {
```

Comment:
I'm not 100% sure but we should be able to simplify all this manual conversion code by declaring `Serializable` structs and just letting wasm-bindgen take care of the conversions. I'll try to come up with some sample code - the docs are pretty nonobvious.